Gems/Atom/RPI/Code/Source/RPI.Public/Pass/AttachmentReadback.cpp

```diff
@@ -416,7 +416,10 @@ namespace AZ
                     AZStd::vector<RHI::SingleDeviceImageSubresourceLayout> imageSubresourceLayouts;
                     imageSubresourceLayouts.resize_no_construct(m_imageDescriptor.m_mipLevels);
                     size_t totalSizeInBytes = 0;
-                    image->GetDeviceImage(RHI::MultiDevice::DefaultDeviceIndex)
+                    // compute lowest device index from mask
+                    int deviceIndex = static_cast<int>(image->GetDeviceMask());
+                    deviceIndex = AZ::Log2(deviceIndex & -deviceIndex);
```

Comment:
Shouldn't the `deviceIndex` computation be implemented in a function somewhere, eg. in `Limits.h`?